### ISSUE TITLE
Where use whisper decoder?

### ISSUE THREAD

**Issue by peggyxpxu** [YuanGongND/ltu#44]
Hi,sir:
    'Whisper Decoder' wav mentioned in ltu-as paper Fig.1. But I don't see whisper decoder being used anywhere. Could you please explain why?
Thank you!

**Comment by YuanGongND** [comment]
yes, we save the output of the decoder (i.e., text) on disk and load it in training. 

you can see how it is used in the inference code https://github.com/YuanGongND/ltu/blob/main/src/ltu_as/inference_gradio.py

**Comment by peggyxpxu** [comment]
> yes, we save the output of the decoder (i.e., text) on disk and load it in training.
> 
> you can see how it is used in the inference code https://github.com/YuanGongND/ltu/blob/main/src/ltu_as/inference_gradio.py

i use https://github.com/YuanGongND/ltu/blob/main/src/ltu_as/inference_batch.py, but not found whisper decoder being used. Is it only used in inference_gradio.py?
